.pylintdict

```diff
@@ -10,6 +10,7 @@ autosummary
 backend
 backpropagation
 bergholm
+bincount
```

Comment:
sklearn no longer takes kwargs and args, so we need to surface their parameters in some of our modules. All of these additions to the pylint dictionary are sklearn variable names.